src/mqueryfront/src/QueryResultsStatus.js

```diff
@@ -142,8 +142,16 @@ class QueryResultsStatus extends Component {
         let progress = Math.floor(
             (this.props.job.files_processed * 100) / this.props.job.total_files
         );
+        let processing = Math.round(
+            (this.props.job.files_in_progress * 100) /
+                this.props.job.total_files
+        );
         let processed =
             this.props.job.files_processed + " / " + this.props.job.total_files;
+
+        if (processing > 0 && processing < 3) {
+            processing = 3;
```

Comment:
This will show `3%` processing even if there's only one file being processed. Comment about minimum with was about style only.

Reply:
(I suggest to remove this if and use max() in `style` below)

Reply:
Sorry, I understood the last review in a weird way and I was confused when implementing it.
New commit has condition that processing > 2 and total_files > 0